dags/taar_weekly.py

```diff
@@ -6,55 +6,77 @@
 from datetime import datetime, timedelta
 from airflow.operators.moz_databricks import MozDatabricksSubmitRunOperator
 from utils.mozetl import mozetl_envvar
+from airflow.operators.subdag_operator import SubDagOperator
+from utils.dataproc import moz_dataproc_pyspark_runner
+from airflow.contrib.hooks.aws_hook import AwsHook
+
+taar_aws_conn_id = "airflow_taar_rw_s3"
+taar_aws_access_key, taar_aws_secret_key, session = AwsHook(
```

Comment:
nit: this fits on one line?

Reply:
Fixed in 4363acb

Reply:
Crap.  I had to revert in 4363acb7 as it broke the DAG definition in https://github.com/mozilla/telemetry-airflow/pull/929/files#diff-00957536695958afe22bc38f1597e7c4R71